dev/dartdoc.dart

```
@@ -56,6 +56,22 @@ dependencies:
     args.add(name.substring(0, name.length - 5));
   }
 
+  [
```

dramatically - it's ok if this script fails fast if its assumptions aren't met